composer.json

```diff
@@ -35,14 +35,22 @@
       "rm -rf base-assets",
       "mv templates/pages/page-style-guide.twig templates/pages/styleguide.html.twig",
       "sed -I '' 's/views/templates/g' assets/css/index.css assets/js/main.js tailwind.config.js templates/organisms/blocks/banner/display/banner.css",
-      "sed -I '' 's/{% include \"/{% include \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/{% include '/{% include '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/{% extends \"/{% extends \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/{% extends '/{% extends '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/featured_image: Image(308)//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/__(//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/, 'ts')/|t/g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/,'ts')/|t/g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig"
+      "sed -I '' 's/{% include \"/{% include \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% include '/{% include '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' 's/{% extends \"/{% extends \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% extends '/{% extends '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ source('/{{ source('@ts_motif\\//g\"   templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ source(\"/{{ source(\"@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% embed '/{% embed '@ts_motif\\//g\"     templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% embed \"/{% embed \"@ts_motif\\//g\"   templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
```

Comment:
I just landed a commit to the WP base theme to turn on a twig namespace for the wordpress theme name. So instead of all this, we could instead update all of the base-assets code to, for example:
```twig
{% include '@thinktimber/organisms/blocks/...' %}
```
And then change this code to simply replace `thinktimber` with `ts_motif` in all files. Thoughts?

Reply:
My thought on that is that it is a dream come true.

Reply:
base-assets part of this change is done!

Reply:
I've pushed an update here, and also discovered a syntax to clean up that horrid directory search hack.